README.md

```diff
@@ -34,6 +34,20 @@ Email us at contact@uqcs.org.au for help, or ask in [Slack](https://slack.uqcs.o
 7. Open your preferred browser.
 8. Navigate to http://localhost:1313/
 
+### Showcasing Your Broject
+
+Have you built something cool you want to show off to the world? It's easy to publish your project to the [UQCS Showcase](https://uqcs.org/showcase/).
+
+1. Fork this repository so you can make changes.
+2. Follow the "Getting Setup" instructions above, replacing the URL with your fork's URL.
+2. Add your project to [content/showcase/_index.html](https://github.com/UQComputingSociety/website/blob/master/content/showcase/_index.html) by copying the format of an existing `showcase-item`. New projects should be added to the top.
+3. Modify the `caption` text with a short blurb for your project. If you want to, you can use basic HTML in this field.
+4. Add an image by adding it to the /img/media/showcase/ folder then specifying the `src` attribute. Images should have an aspect ratio of 1000x750.
```

Comment:
Please change the `x` to `&times;`.  
(Possibly also add mention of minimum size; larger images of that ratio are allowed.)